src/containers/apps/Apps.tsx

```diff
@@ -23,13 +24,16 @@ export default class Apps extends ApiComponent<
                   projects: ProjectDefinition[]
               }
             | undefined
+        showCreateAppForm: boolean
     }
 > {
     constructor(props: any) {
         super(props)
         this.state = {
             isLoading: true,
             apiData: undefined,
+            showCreateAppForm:
+                StorageHelper.getCreateAppFormVisibilityFromLocalStorage(),
```

Comment:
Definitely we don't want to store this in local storage. It's just a "state". Clicking back and forth should "reset" this state.